lib/Test2/Harness/Collector.pm

```diff
@@ -271,6 +281,11 @@ sub jobs {
         );
     }
 
+    # The collector didn't read in all the jobs because it'd run out of file handles. We need to let the stream know we're behind.
+    if(scalar %$jobs >= $max_jobs_to_process) {
+        print STDERR "WARN: The Yath Collector is running behind. More than $max_jobs_to_process test results have not been processed.\n";
```

Comment:
This looks good in the output stream:

```
( PASSED )  job 273    t/test1.t
( PASSED )  job 361    t/test2.t
WARN: The Yath Collector is running behind. More than 300 test results have not been processed.
( PASSED )  job 267    t/test3.t
( PASSED )  job 279    t/test4.t
( PASSED )  job 366    t/test5.t
```
Is there a better way to pass this message to the formatter as a WARN? I see other random prints so I know it's not unheard of to do this?

Reply:
See my additional commit which turns this into a proper events as you requested.

Reply:
Yep! thank you!